.github/workflows/cron-2022.4-core-ubuntu-latest.yaml

```diff
@@ -10,8 +10,8 @@ env:
 on:
   workflow_dispatch: {}
   schedule:
-    # dev - every 2 hours
-    - cron: '37 */2 * * *'
+    # dev - every 4 hours, starting at the zeroth hour
```

Comment:
I like this - every 2 hrs was feeling a bit much. Also, being able to run the jobs on-demand using `workflow_dispatch` is a nice escape hatch for time-sensitive situations.